make manifests returns "Error: no kustomization.yaml file under /tmp/kustomize-490795293/config"

The manifest target fails with

````
[rwsu@localhost-live cluster-api-provider-baremetal]$ make manifests
go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go all
CRD manifests generated under '/home/rwsu/go/src/github.com/metal3-io/cluster-api-provider-baremetal/config/crds' 
RBAC manifests generated under 'config/rbac' 
webhook manifests generated under '/home/rwsu/go/src/github.com/metal3-io/cluster-api-provider-baremetal/config/webhook' directory
kustomize build config/ > provider-components.yaml
2020/02/03 13:54:50 Adding nameprefix and namesuffix to Namespace resource will be deprecated in next release.
echo "---" >> provider-components.yaml
cd vendor && kustomize build github.com/openshift/cluster-api/config >> ../provider-components.yaml
Error: no kustomization.yaml file under /tmp/kustomize-212061250/config
make: *** [Makefile:40: manifests] Error 1

[rwsu@localhost-live cluster-api-provider-baremetal]$ go version
go version go1.13.6 linux/amd64
[rwsu@localhost-live cluster-api-provider-baremetal]$ cat /etc/redhat-release 
Fedora release 31 (Thirty One)
[rwsu@localhost-live cluster-api-provider-baremetal]$ kustomize version
Version: {KustomizeVersion:1.0.11 GitCommit:8f701a00417a812558a7b785e8354957afa469ae BuildDate:2018-12-04T18:42:24Z GoOs:unknown GoArch:unknown}
````

--- a/Makefile
+++ b/Makefile
@@ -37,7 +37,7 @@ manifests:
 	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go all
 	kustomize build config/ > provider-components.yaml
 	echo "---" >> provider-components.yaml
-	cd vendor && kustomize build github.com/openshift/cluster-api/config >> ../provider-components.yaml
+	kustomize build vendor/github.com/openshift/cluster-api/config >> provider-components.yaml
 
 # Run go fmt against code
 fmt:

